pages/api/submissions/[id]/approvals.ts

```diff
@@ -0,0 +1,50 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import StatusCodes from 'http-status-codes';
+import { approveSubmission, returnForEdits } from 'lib/submissions';
+import { isAuthorised } from 'utils/auth';
+import { notifyReturnedForEdits } from 'lib/notify';
+
+const handler = async (
+  req: NextApiRequest,
+  res: NextApiResponse
+): Promise<void> => {
+  const { id } = req.query;
+
+  const user = isAuthorised(req);
+
+  switch (req.method) {
+    case 'POST':
+      {
+        const submission = await approveSubmission(
+          String(id),
+          String(user?.email)
+        );
+        res.json(submission);
+      }
+      break;
+    case 'DELETE':
```

Comment:
Why is it using DELETE, shouldn't this be a PATCH?

Reply:
it just uses the method to decide what to do - this is at an `/approvals` sub-route, so POSTing or DELETing your approval makes semantic sense to me